### PR TITLE
Fix the height of new post text area and add scrollbar

### PR DESCRIPTION
**Pull request by aviskarkc10**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Currently because of the auto resizing of new post textarea, if a post is long, whatever you are typing is not visible. This PR intends to fix that. 

## Related Tickets & Documents
#6752 

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

Peek 2020-04-23 21-10

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Comment by aviskarkc10** [comment]
@rhymes This is how I have decided to solve this issue. I have a reservation though, because I am not sure where else `bodyMarkdown.jsx` file is used and where else it might impact things.

I saw that `BodyMarkdown` component is also used in `dev.to/app/javascript/listings/listingForm.jsx` file, but I couldn't really figure out where it would be rendered.

**Comment by joshpuetz** [comment]
Hi @aviskarkc10, thanks so much for your contribution and apologies on the delay in reviewing it. We'll take a look in the next few days and provide a review.

In the meantime, please resolve the conflict in `app/assets/stylesheets/preact/article-form.scss`. 

Thanks so much!

**Comment by aviskarkc10** [comment]
No apology needed at all. Will fix the conflict.

**Comment by aviskarkc10** [comment]
@joshpuetz I merged my code with master but now I get a bunch of eslint errors. Should I just go ahead with the `--no-verify` flag? The errors are in files that I haven't touched and are a lot.

eslint_errors

**Comment by joshpuetz** [comment]
Are you getting those errors when trying to commit @aviskarkc10? Pinging @nickytonline for some feedback here...

**Comment by nickytonline** [comment]
I merged master for you but it's showing a tonne of other file changes now @aviskarkc10. Let me sort it out. 🙃

**Comment by nickytonline** [comment]
@aviskarkc10, I had to end up forcing a push to go back to your two initial commits. I'm still not sure why merging master initially resulted in adding 44 extra files to your PR.

Once you've fixed the conflict, you can merge the latest of master and when you commit it, add `--no-verify` to the end of the commit command, i.e. `git commit -m"Merge remote-tracking branch 'origin/master' into aviskarkc10/new-post-textarea-height-6752" --no-verify`.

**Comment by aviskarkc10** [comment]
@nickytonline I have pushed my changes.

**Comment by Ridhwana** [comment]
Pinging @ludwiczakpawel here too as this will need to be tested on the new editor layout when merged.

**Comment by ludwiczakpawel** [comment]
This definitely fixes our editor but we're currently working on [new version of the editor](https://github.com/thepracticaldev/dev.to/pull/7524), where this issue has been addressed already. I'm ok with merging it to master now - but FYI new editor will be live any day soon.

File: app/assets/stylesheets/preact/article-form.scss [hunk 1]
```diff
@@ -361,21 +361,19 @@
...
-  // scrollbars not needed because the preact-textarea-autosize package
-  // automatically resizes the article form
-  overflow: hidden;
   background: var(--card-bg);
   color: var(--card-color);
+  overflow-y: auto;
   @media screen and (min-width: 600px) {
     font-size: 18px;
-    min-height: calc(85vh - 280px);
+    height: calc(85vh - 200px);
```
Comment (rhymes) [inline app/assets/stylesheets/preact/article-form.scss:376]: Hi @aviskarkc10, it looks great! just a small suggestion here, for big screens:
 
```suggestion
    height: calc(82vh - 200px);
```

before:

Screenshot_2020-05-14 New Post - DEV(local) Community 👩‍💻👨‍💻(1)

after:

Screenshot_2020-05-14 New Post - DEV(local) Community 👩‍💻👨‍💻

Comment (aviskarkc10) [reply app/assets/stylesheets/preact/article-form.scss]: @rhymes I have added the change

**Comment by nickytonline** [comment]
Thanks so much for your contribution @aviskarkc10! 👏

**Comment by aviskarkc10** [comment]
No problem at all :smile:

**Pull request merged**